generators/service-alert-notification/templates/python/requirements.txt

```diff
@@ -1 +1 @@
-requests==2.13.0
+requests
```

Comment:
[`service-push` still uses `requests==2.13.0`](https://github.com/ibm-developer/generator-ibm-service-enablement/blob/b99960ebc3dfaac53502112b8419cec45a845823/generators/service-push/templates/python/requirements.txt). If both of them are used in the same project, is that going to be a problem since the dependencies aren't the same (i.e. will it be put into requirements/Pipfile twice)? Are there other `requests` dependencies that need to change?